CHANGES.rst

```diff
@@ -124,6 +124,8 @@ refpix
 
 - Fixed bugs in PR #4575; added unit tests [#4596]
 
+- Changed the data type of columns OUTPUT and ODD_EVEN in DQ [#4618]
```

Comment:
This is in the `IRS2Model` reference file DQ?  Might be good to indicate that here.

Reply:
Yes, good suggestion.